lib/l10n/generated/app_localizations.dart

```diff
@@ -0,0 +1,404 @@
+import 'dart:async';
```

Comment:
I’m not saying this is necessarily bad practice, but please consider whether we should commit generated files to git.

Each approach has its own pros and cons, and there are quite a few discussions about this online.

https://medium.com/steeple-product/flutter-should-we-track-our-generated-files-b9528b907b2